l5kit/l5kit/geometry/transform.py

```diff
@@ -111,6 +112,8 @@ def transform_points(points: np.ndarray, transf_matrix: np.ndarray) -> np.ndarra
     Returns:
         np.ndarray: array of shape (N,2) for 2D input points, or (N,3) points for 3D input points
     """
+    if len(points.shape) == 1:
```

Comment:
Why was this changed?

Reply:
because we're deprecating the function `transform_point` and this one will replace it. I should probably ensure the final shape is preserved though, what do you think?

Reply:
I've removed this from the PR. I'll start a dedicated one targeting this one